widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageView.kt

```diff
@@ -220,7 +220,12 @@ class MessageView(
         updateSendButtonState(state.sendMessageButtonState)
         updateSendMessageError(state.showMessageLimitError)
         updateMessageEditText(state.messageEditTextEnabled, state.showMessageLimitError)
-        addAttachmentButton.isEnabled = state.addAttachmentButtonEnabled
+        if (state.addAttachmentButtonVisible) {
+            addAttachmentButton.visibility = VISIBLE
+            addAttachmentButton.isEnabled = state.addAttachmentButtonEnabled
+        } else {
+            addAttachmentButton.visibility = GONE
+        }
```

Comment:
Do we need to execute `addAttachmentButton.isEnabled = state.addAttachmentButtonEnabled` for the `else` case as well?

Reply:
Just optimization. The button is invisible, so it doesn't matter if is it enable or not.